src/main/java/com/github/manolo8/darkbot/gui/MapDrawer.java

```diff
@@ -224,4 +266,72 @@ public double toGameLocationY(int screenY) {
             return (screenY / (double) getHeight()) * mapBounds.getHeight();
         }
     }
+
+    private static class BackgroundImgDownloaderTask extends SwingWorker<Image, Void> {
```

Comment:
This should probably be part of GameResourcesAPI (the same that handles translations, should have the methods to search for images)